File: pync.c
#include "pync.h"

static int counter = 0;

void
increment (void)
{
   counter++;
}

void
decrement (void)
{
   counter--;
}

void
reset (void)
{
   counter = 0;
}

int
get_number (void)
{
   return counter;
}

File: pync.py
import pync

for _ in range(256):
   pync.increment()

print(f"[PYNC.PY] INCREMENT LOOP: {pync.get_number()}")

pync.increment()
pync.increment()
print(f"[PYNC.PY] INCREMENT TWICE: {pync.get_number()}")

pync.decrement()
print(f"[PYNC.PY] DECREMENT: {pync.get_number()}")

pync.reset()
print(f"[PYNC.PY] RESET: {pync.get_number()}")
